pages/Documents/Security&Authentication/ConsumerAuthentication/configuration.md

```diff
@@ -21,33 +21,40 @@ indicator: both
 
 ![Search for Consumer Identity Provider in the Management Console](img/mgmt_search_consumer_idp.png)
 
+![Consumer Identity Provider in the Management Console](img/mgmt_consumer_idp.png)
+
+
 {:start="3"}
 3.  Click on 'Click to configure'
```

Comment:
I know, this issue existed before, but let’s fix the inconsistency and switch to double quotes here. Thanks!